uber/templates/emails/shifts/schedule.html

```diff
@@ -5,23 +5,28 @@
 {{ attendee.first_name }},
 <br/> <br/>
 
-{% if attendee.badge_type == c.STAFF_BADGE %}    
+{% if attendee.badge_type == c.STAFF_BADGE %}
 Thanks again for Staffing at {{ c.EVENT_NAME }}!
 {% endif %}
-    
+
 {% if c.VOLUNTEER_RIBBON in attendee.ribbon_ints %}
 Thanks again for Volunteering at {{ c.EVENT_NAME }}!
 {% endif %}
 <br/> <br/>
 
-{% if attendee.badge_type == c.STAFF_BADGE %}   
-- Please remember to also pick up your TWO {{ c.EVENT_NAME }} Staff Shirts from the MAGFest MERCH Booth when you get to {{ c.EVENT_NAME }}.  As a reminder, there is an expectation that you will wear one of these two shirts while working your scheduled shifts during {{ c.EVENT_NAME }}, (Thurs-Sunday, not during load-in or load-out).  When not on shift, you should wear a different shirt, so you can enjoy MAGFest 2017 with everyone else.    
+{% if attendee.badge_type == c.STAFF_BADGE and (c.SHIRTS_PER_STAFFER > 0 or c.STAFF_ELIGIBLE_FOR_SWAG_SHIRT) %}
```

Comment:
This is a minor nitpick, but don't forget that as an HTML email, we can space things out a bit. It's kind of a mass of text like this. :'D

Reply:
This is totally a personal preference thing, so I can go either way on it, but I prefer breaking text up over multiple lines instead of having one really long horizontal line.

Reply:
Yeah, same! But you've only used single linebreaks here, so the text as a whole is still 'crowded' feeling since it looks like one big paragraph despite having multiple if-conditions and stuff. Totally a personal preference thing though.

Reply:
Oh snap! You're right!